src/_shared/js/capi-multiple.js

```diff
@@ -3,16 +3,10 @@ import { getIframeId, getWebfonts, resizeIframeHeight } from
 import { write } from './dom.js';
 import { enableToggles } from './ui.js';
 import { insertImage } from './capi-images.js';
+import { setEditionLink } from './ads';
 
 const ENDPOINT = 'https://api.nextgen.guardianapps.co.uk/commercial/api/capi-multiple.json';
```

Comment:
Yeah I was originally using that, but didn't want to run frontend locally after I merged the endpoint PR. Can add it back in if you think it would help with future testing?